internal/app/config.go

```diff
@@ -39,6 +40,23 @@ type Auth0Config struct {
 	WebClientID  string
 }
 
+type AuthConfig struct {
+	Domain string `default:"http://localhost:8080"`
+	Key    string
```

Comment:
Not used
```suggestion
```

Reply:
It's used [here ](https://github.com/reearth/reearth-backend/blob/f493cd30062bf473703ba78ef50cb5a39cd5c979/internal/app/auth_server.go#L34)

Reply:
Sorry i forgot it